### DISCUSSION TITLE
Fix the time column while scrolling the table to the right.

### DISCUSSION THREAD

**Discussion by ravi7248** [grafana/grafana#42340] [Dashboards, Panels & Visualizations]
The first column **time** should be fixed or there should one option to fix the time column. Sometimes no of the columns are many so we have to scroll to the right for seeing the value but the time column gets hidden then we have to come back again and check the time and go back to a particular column.

Sometimes, it's difficult to manage when we have too many rows and columns. This is the basic feature of a table.
Like in excel we can freeze the first column and the first row. In the same way, there should be some option to freeze the time column.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
